benchmark/utils/CxxTests/CreateObjects.h

```diff
@@ -0,0 +1,3 @@
+struct CxxIntBox {
+  int x;
```

Comment:
In the standard library, we'd usually call the member "value" in similar situations.